style_guide.md

```diff
@@ -0,0 +1,309 @@
+# Jsonnet Style Guide
+This style guide documents how we want to handle Jsonnet files when building questionnaire schemas with consistency. For more general principles and styling rules go to [Databricks repository](https://github.com/databricks/jsonnet-style-guide#databricks-jsonnet-guide).
+
+## Folder structure
+- The following structure is used:
+
+    ```
+    england-wales/
+       lib/
+          common_rules.libsonnet
+          placeholders.libsonnet
+       <questionnaire>/
+          <questionniare>.jsonnet
+          lib/
+             rules.libsonnet
+          blocks/
+             <section-one>/
+                <block-one>.jsonnet
+                <block-two>.jsonnet
+             <section-two>/
+    northern-ireland/   
+    ```
+- Jsonnet source files are separated into two top-level folders - `england-wales` and `northern-ireland` 
+- The `lib/` folder contains common Jsonnet used across all questionnaire schemas
+  - The file extension should be `.libsonnet`
+  - Each file is grouped by function, currently `common_rules` and `placeholders`
+- Each questionnaire has it's own folder, within this:
+  - A top-level questionnaire definition Jsonnet file 
+  - A `lib/` folder holds common Jsonnet used across multiple blocks
+    - The file extension should be `.libsonnet`
+    - Each file here can extend common Jsonnet from the questionnaire level lib folder
```

Comment:
This is a repeat of line 24-26. Just mention "A `lib/` folder"?

Reply:
Possibly. The key point that needs to be made here is the second one.

Reply:
Removed first mention, thanks!👍